docs/manual/working/scalaGuide/main/json/ScalaJsonAutomated.md

```diff
@@ -34,13 +34,15 @@ The above example can be made even more concise by using body parsers with a typ
 
 ### Requirements
 
-These macros rely on a few assumptions about the type they're working with :
-- It must have a companion object having `apply` and `unapply` methods
+These macros support classes and sealed traits. It relies on the following assumptions.
```

Comment:
This section is a bit confusing because it does not apply to sealed traits. Might be better to say "The macros automatically work for classes meeting following requirements:"

Reply:
"The macros ~~automatically~~ for classes and traits meeting the following requirements" ?

Reply:
"The macros work for classes and traits meeting the following requirements"